modules/tinymce/src/core/main/ts/api/Mode.ts

```diff
@@ -82,14 +81,17 @@ export const create = (editor: Editor): EditorMode => {
       editorReadOnly: true
     }
   });
+  const activeMode = Cell<[ string, EditorModeApi ]>([ 'design', availableModes.get().design ]);
```

Comment:
I'm a little confused by this change. Could we just keep it as a string?

Reply:
ah I didn't get the github update before posting my review. Yeah it's leftover from the API change.

Reply:
Yeah, this is no longer required. I'll revert all the changes to the mode anyway since the permissions will not be implemented in 7.4. We can investigate this again to prevent the integrator from switching modes in future versions.